lib/ssh_scan/database.rb

```diff
@@ -0,0 +1,26 @@
+# API Databases
+require 'ssh_scan/database/mongo'
+require 'ssh_scan/database/sqlite'
+
+module SSHScan
+  class DatabaseConfig
+    def initialize(opts = {})
+      @@db_type = opts['database']['type'] || 'mongodb'
```

Comment:
You will also get NoMethod errors if the opts variable doesn't contain the 'database' key.  This is a blocking bug.

Example:

    2.2.3 :002 > opts = {}
     => {} 
    2.2.3 :003 > opts['database']['type']
    NoMethodError: undefined method `[]' for nil:NilClass
    	from (irb):3
    	from /Users/jclaudius/.rvm/rubies/ruby-2.2.3/bin/irb:15:in `<main>'